socorro/cron/jobs/bugzilla.py

```diff
@@ -36,6 +36,31 @@
 )
 
 
+def find_signatures(content):
+    """Return a list of signatures found inside a string.
+
+    Signatures are found between `[@` and `]`. There can be any number of them
+    in the content.
+
+    Example:
+    >>> "some [@ signature] [@ another_one] and [@ even::this[] ]"
```

Comment:
This needs to be passed to `find_signatures` like this:

```
>>> find_signatures("some [@ signature] [@ another_one] and [@ even::this[] ]")
set(['signature', 'another_one', 'even::this[]'])
```

Also, does this work only on strings? What happens with unicode? Is it ever possible to get a unicode signature?